dist/common/scripts/scylla_io_setup

```diff
@@ -232,6 +232,47 @@ if __name__ == "__main__":
                 disk_properties["read_bandwidth"] = 507338935 * nr_disks
                 disk_properties["write_iops"] = 57100 * nr_disks
                 disk_properties["write_bandwidth"] = 483141731 * nr_disks
+            elif idata.instance_class() in ("c6gd", "m6gd", "r6gd", "x2gd"):
+                if idata.instance_size() == "medium":
+                    disk_properties["read_iops"] = 14808
```

Comment:
We could simplify the code by creating a large dict of dict of namedtuple and then select from the dict:

```python
known_instance_io = {
    'i3': {
         'large': iorates(read_iops: ..., write_iops: ..., read_bw: ..., write_bw: ...)
         'xlarge': ///
    },
    'i3en': {
         ...
```

Of course, not suggesting to do it now.